### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by ccris02**
|	Site	|	Urlscan	|
|	----------------------------------------------	|	----------------------------------------------	|
|	dappsvalidatorsync.com	|	https://urlscan.io/result/60422bbb-b614-4337-a496-b3b0fe7cb7b6/	|
|	deficryptoconnects.com	|	https://urlscan.io/result/615b2f87-2ba8-495e-9a66-f146b8fc8344/	|
|	syncconnectrectifier.com	|	https://urlscan.io/result/7e974ffc-6896-4eda-8e2d-010023a13527/	|
|	webdappauth.com	|	https://urlscan.io/result/0eb12404-c277-4ebf-bf3e-d1980ebbe553/	|
|	syncdappscnetfix.com	|	https://urlscan.io/result/4ba5962b-d5b8-479d-bc87-f5c81af10bfc/	|
|	tokenpocketn.cc	|	https://urlscan.io/result/f56ff832-5623-4d33-9273-8e5a0a2a0695/	|
|	walletsafefix.com	|	https://urlscan.io/result/2e185d90-38f0-41e2-b543-ed747e8b55ed/	|
|	imtokenzh88.com	|	https://urlscan.io/result/1a6cd23c-1d86-4042-b0fe-ee891c23a9d0/	|
|	qcxdapps.com	|	https://urlscan.io/result/c65cd120-913d-4ae3-9f7c-08f7662c01d3/	|
|	eng-sync-protocol.com	|	https://urlscan.io/result/ce264cb9-a66b-48e8-abb1-1c940aa5a72f/	|
|	dappscnet.online	|	https://urlscan.io/result/629f511b-94b8-446d-8d16-f98ebad60667/	|
|	manualonline-sync.com	|	https://urlscan.io/result/6a0c39c4-2f10-4669-87ff-d1d04d6a67c3/	|
|	dexconnectposts.com	|	https://urlscan.io/result/c66b178f-2689-4242-8dce-6b361eed626c/	|
|	walletsbrowser.com	|	https://urlscan.io/result/e0a445b6-d8ee-49fd-a93f-1b4b6f2878c6/	|

### CODE REVIEW

**Review by jacogr — APPROVED** [review]

File: all.json [hunk 1]
```diff
@@ -2,6 +2,20 @@
 	"allow": [],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"dappsvalidatorsync.com",
+		"deficryptoconnects.com",
```
Comment (jacogr) [inline all.json:6]:
```suggestion
```

Dupe.

File: all.json [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+		"syncconnectrectifier.com",
+		"webdappauth.com",
+		"syncdappscnetfix.com",
+		"tokenpocketn.cc",
+		"walletsafefix.com",
```
Comment (jacogr) [inline all.json:11]:
```suggestion
```

Dupe.

File: all.json [hunk 3]
```diff
@@ -2,6 +2,20 @@
 	"allow": [],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"dappsvalidatorsync.com",
+		"deficryptoconnects.com",
+		"syncconnectrectifier.com",
+		"webdappauth.com",
```
Comment (jacogr) [inline all.json:8]:
```suggestion
```

Dupe.

**Review by jacogr — COMMENTED** [review]
Dupes as per https://github.com/polkadot-js/phishing/runs/6001812369?check_suite_focus=true

File: all.json (hunk 3 above)
Comment (frankywild) [reply all.json]: Deleted some more duplicated.

"deficryptoconnects.com",
 "walletsafefix.com",
  "webdappauth.com"

**Review by frankywild — APPROVED** [review]

**Pull request merged**